src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -107,7 +111,7 @@ type private Sum() =
 
     member this.Final(_aggContext, _anonContext) = state
 
-type private DiffixCount(aidsCount) =
+type private DiffixCount(aidsCount, requestedOutput) =
```

Comment:
The language already provides a standard mechanism for partially overriding an object's behavior: inheritance. Isn't that sufficient? I think it would be more elegant than a constructor flag.

Reply:
Inheritance is exactly what I was trying to avoid, until we get a sense of the abstractions required here (e.g. for the optimized way of handling the computation). I feel like it would be pretty likely that those abstractions would need to be redesigned, but we can do via inheritance as well if you prefer.

Reply:
I don't think I understand what you mean. In my mind, the abstraction already exists (`IAggregator`) and I suggest we keep using it as it is.

Reply:
To be clear: I suggest that `CountNoise` inherits from `Count` and only overrides the `Final` function.